src/foam/u2/view/UnstyledTableView.js

```diff
@@ -411,7 +418,7 @@ foam.CLASS({
 
           //with this code error created  slot.get cause promise return
           //FIX ME
-          return this.slot(function(data, data$delegate, order, updateValues) {
+          return this.slot(function(data, data$delegate, onDAOUpdate, order, updateValues) {
```

Comment:
Is there a better way to do this? Using data$proxy instead, maybe?

Reply:
hmmm, probably, I don't have any suggestions as for now though. for some reason data$proxy doesn't work

Reply:
we're using this approach on TableView